service/src/main/java/com/zebrunner/reporting/service/feign/FeignConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.zebrunner.reporting.service.feign;
+
+import feign.Request;
+import feign.Retryer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+public class FeignConfiguration {
+
+  @Bean
+  public Request.Options timeoutConfiguration(){
+    return new Request.Options(5000, TimeUnit.MILLISECONDS,
+            30000, TimeUnit.MILLISECONDS, false);
+  }
+
+  @Bean
+  public Retryer retryer() {
+    return new Retryer.Default(1000, 5000, 3);
```

Comment:
I wonder what those values mean. Is there any way to make this more explicit allowing to understand what's going on without inspecting source code? add comment maybe